src/mod_openssl.c

```diff
@@ -546,6 +551,163 @@ network_openssl_ssl_conf_cmd (server *srv, plugin_config *s)
   #endif
 }
 
+#if defined(HAVE_WOLFSSL_SSL_H) && defined(HAVE_PK_CALLBACKS)
+
+static int tls_create_key_cb(WOLFSSL* ssl, ecc_key* key, word32 keySz,
+    int ecc_curve, void* ctx)
+{
+    int     ret;
+    server* srv = (server*)ctx;
+    WC_RNG  rng;
+
+    (void)ssl;
```

Comment:
Please use the UNUSED() macro for unused variables instead of explicitly casting to (void). In the future, the UNUSED() macro might include __attribute__((unused)) for compilers which support that attribute.